e2e/pierced-react/README.md

```diff
@@ -3,7 +3,6 @@
 To run the app (that shows the server side piercing) you need to run (in order):
  - `pnpm i` at the root of the monorepo
  - `pnpm build` in /reframed
```

Comment:
can you please update these steps in the following style:
```suggestion
 - `pnpm --filter reframed build`
 - `pnpm --filter ./packages/web-fragments build`
 - `pnpm --filter ./e2e/pierced-react-remix-fragment build && pnpm --filter ./e2e/pierced-react-remix-fragment start`
 - new terminal: `pnpm --filter ./e2e/pierced-react dev`
```

Reply:
what about:
```
 - `pnpm i`
 - `pnpm --filter reframed --filter ./packages/web-fragments --filter ./e2e/pierced-react-remix-fragment build`
 - `pnpm --filter ./e2e/pierced-react-remix-fragment start`
 - in a new terminal: `pnpm --filter ./e2e/pierced-react dev`
```
?

🙂

Reply:
that works! thanks

Reply:
(works much better)